lib/commands.js

```diff
@@ -75,7 +63,9 @@ function list () {
     paths.map(fsReadDir)
   )
     .then((results) => {
-      return filterExe(filterBlinkm([].concat(...results)));
+      let mergedResults = [];
+      mergedResults = mergedResults.concat.apply(mergedResults, results);
```

Comment:
`results` is an Array of Arrays of Strings. If I've written this properly, `mergedResults` ends up being a flattened Array of Strings.

Very annoying seeing 1 line blown into 3 just because Node.js 4.x doesn't support the spread operator. :S